Created flake for project

inkpot-monkey opened the pull request:
Pretty simple project to package as everything is already contained within the repository.

There are some apps that load the browser version and some addition checks that make use of the general tests in the project

inkpot-monkey committed:
Created flake for project

inkpot-monkey commented:
Sorry this was too eager, need to get this check by the ngi review team

Ayms commented:
Yes super simple to package, the only external thing is the third party check for the code loaded https://github.com/Ayms/bitcoin-transactions/blob/master/hash.txt (not so external right now because it is myself)

inkpot-monkey committed:
Use current directory as source

inkpot-monkey committed:
Project has no versioning

inkpot-monkey committed:
Made not about usage of wallet-clear

inkpot-monkey committed:
Add documentation about the flake apps

inkpot-monkey committed:
Fixed narhash being out of date

inkpot-monkey committed:
Feedback changes

1. src now equals lib.cleanSource ./.
2. supportedSystems expanded to any system that can run a browser
3. combined apps into single script with flags
4. created nixosModule which creates a .desktop file
5. removed unused version var
6. enclose bash vars in ""

inkpot-monkey committed:
extract html into var

inkpot-monkey committed:
Version now equals last git commit

inkpot-monkey committed:
Src = self

inkpot-monkey committed:
Convert ShellScriptBin to ShellScript

inkpot-monkey committed:
simplify inherit

Co-authored-by: Aaron Janse <aaron@ajanse.me>

inkpot-monkey committed:
Refactor tests

inkpot-monkey committed:
make app defaultApp

inkpot-monkey committed:
Make use of `makeDesktopItem` function

inkpot-monkey committed:
Tidy

inkpot-monkey commented:
Hi,

This flake has been completed and integrates this project into the Nix ecosystem :0

It builds a package and exposes a shell command
`nix run  github:Ayms/bitcoin-transactions` which opens the project with xdg-open

it also exposes a flag (clear || c) which runs the walletclear.html 

and makes it installable as a desktop item on NixOS through a nixosModule (which is why the png has been added to the repo as well)
